Problem-198

```diff
@@ -0,0 +1,24 @@
+/*
+Time Complexity: O(n)
+Did not run in leetcode: 39 / 69 test cases passed.
+Approach: The only solution i could think of was that once check odd positions and once check even positions.
+I understand that this is totaly incorrect, but i could not come up with an algorithm to accomodate best houses.
+
```

Comment:
try finding the max you can get from any given 2 neighbours at a time and keep moving forward